src/Header/MobileNavigation/MobileNavElement/MobileNavElement.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import '../../../Styles/MobileNavElement.scss';
+import { AnimateOnChange } from 'react-animation';
+import Collapse from '@material-ui/core/Collapse';
+import lists from '../../../ExpandedListItems.json';
+import MobileExpandedList from './MobileExpandedList/MobileExpandedList';
+
+const MobileNavElement = (props) => {
+  const [isExpanded, setIsExpanded] = useState(false);
+
+  let chosenList;
+  Object.keys(lists.list).forEach((key) => {
+    if (key === props.name) {
+      chosenList = key;
+    }
+  });
+
+  const handleMouseClick = () => {
+    setIsExpanded(!isExpanded);
+  };
+
+  return (
+    <>
+      <div onClick={handleMouseClick} role="presentation" className="MobileNavElement">
```

Comment:
Your class names style varies a lot. Look up CSS style guide. As a rule of thumb - use only **lower case** and hyphens (-). You *can* use BEM if you want to - it plays nicely with SCSS.